src/consensus/pool/parent_ready_tracker.rs

```diff
@@ -249,9 +250,9 @@ mod tests {
     #[test]
     fn out_of_order_notars() {
         assert_eq!(Slot::genesis().slots_in_window().count(), 4);
-        let block1 = (Slot::new(1), [1; 32].into());
-        let block2 = (Slot::new(2), [2; 32].into());
-        let block3 = (Slot::new(3), [3; 32].into());
+        let block1 = (Slot::new(1), Hash::random_for_test().into());
```

Comment:
we have a lot of `Hash::random_for_test().into()`.  We could maybe introduce `BlockHash::random_for_test()` in a future PR.  It will be fewer characters and more readable (especially when one is looking at code outside of an IDE).